zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -85,6 +86,7 @@
   public static Server jettyWebServer;
   public static NotebookServer notebookWsServer;
   public static Helium helium;
+  public static UsersRepo usersRepo;
```

Comment:
Maybe `usersRepo` does not best describe what it is right now or what it's going to be in the future. I'm not a fan of making a lot of comment in the source code, but in this case, a just simple comment that explains the purpose of this variable may help other developers understand the intention.